### PR TITLE
Add support for ip on webserver

### PR DESCRIPTION
**Pull request by jlsjonas**

### CODE REVIEW

**Comment by benbrown** [comment]
I think this value should go into the configuration, not as a separate parameter.

**Comment by jlsjonas** [comment]
@benbrown you're right, changes reflected in last commit

File: lib/SlackBot.js [hunk 1]
```diff
@@ -209,6 +209,7 @@ function Slackbot(configuration) {
             static_dir = slack_botkit.config.webserver.static_dir;
 
         slack_botkit.config.port = port;
+        slack_botkit.config.ip = slack_botkit.config.ip || 127.0.0.1;
```
Comment (anonrig) [inline lib/SlackBot.js]: `127.0.0.1` should be a string. What do you think @jlsjonas ?

**Comment by LoneRifle** [comment]
Just to help out with paperwork, this issue will resolve #159

**Comment by anonrig** [comment]
@LoneRifle We're waiting a response from @jlsjonas due to a bug on the code. Will soon asap when its resolved.

**Comment by LoneRifle** [comment]
I notice this only takes care of slack and not fb messenger. If need be, I wouldn't mind submitting a PR that encompasses the above changes (and reflects @jlsjonas authorship of course) with the suggested bugfix as well as equivalent ones for FB. fwiw, i'm more interested in the latter. 

I'll come back to this thread in about a month and see where we should go.

**Comment by benbrown** [comment]
I have added support for the `hostname` configuration option to _all_ of the botkit connectors.

Note it is  `hostname` not `ip` since this is what the parameter is named in the Express documentation.

**Pull request closed without merging**